sympy/assumptions/ask.py

```diff
@@ -68,6 +68,22 @@ def antihermitian(self):
         # TODO: Add examples
         return Predicate('antihermitian')
 
+    @predicate_memo
+    def idempotent(self):
+        """
+        Idempotent predicate.
+
+        ``Q.idempotent(x)`` is true iff ``x^n=x`` for positive ``n``.
+
+        References
+        ==========
+
+        .. [1] https://en.wikipedia.org/wiki/Idempotence
+
+        """
+        # TODO: Add examples
```

Comment:
You've left a TODO here. Should that be done now?

Reply:
That's a copy-pasting error from the docs for Hermitian and AntiHermitian just above. I have deleted it.